x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -5,17 +5,9 @@
  * 2.0.
  */
 
-/* eslint-disable @kbn/eslint/no-restricted-paths */
-import { rawRules } from '../../server/lib/detection_engine/rules/prepackaged_rules/index';
 import { getMockThreatData } from '../../public/detections/mitre/mitre_tactics_techniques';
 import { getTimeline, CompleteTimeline, getIndicatorMatchTimelineTemplate } from './timeline';
 
-export const totalNumberOfPrebuiltRules = rawRules.length;
-
-export const totalNumberOfPrebuiltRulesInEsArchive = 127;
```

Comment:
what's EsArchive? is that something we still need?

Reply:
if not, i'm glad to see this go

Reply:
It looked like a dead variable, so out it goes! :-)